Src/CSharpier.Tests/TestFiles/ObnoxiousEdgeCases/ObnoxiousEdgeCases.cst

```diff
@@ -2,15 +2,13 @@
 // try to clean these up if you are modifying formatting that affects them.
```

Comment:
Is this comment still necessary now that you've cleaned these up, or do you have more cleanup you intend to do?

Reply:
Just glancing at them, there are probably still some improvements to be made. I mostly have the comment here so that people don't think this is really how we want things to look. If I ever get around to #272 this file can go away completely and they'll turn into proper test cases